x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row_messages_pane.tsx

```diff
@@ -52,8 +51,14 @@ export const ExpandedRowMessagesPane: FC<Props> = ({ analyticsId }) => {
       }
     };
   };
-
   useRefreshAnalyticsList({ onRefresh: getMessagesFactory() });
 
-  return <JobMessages messages={messages} loading={isLoading} error={errorMessage} />;
+  return (
+    <JobMessages
+      messages={messages}
+      loading={isLoading}
+      error={errorMessage}
+      refreshMessage={() => getMessagesFactory()()}
```

Comment:
Is there a reason `refreshMessage` is not just set to `getMessagesFactory()` ? 
If `getMessageFactory()` just returns a function then you should be able to just pass it that function through and it should be called correctly. I might be missing some context here.

Reply:
Updated in https://github.com/elastic/kibana/pull/67750/commits/9381a0d81beefa02ec6307a6fa22a797bed58197